blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -89,6 +89,8 @@ Changes
 
 - Added left and right scalar functions.
 
+- Added timezone scalar function.
```

Comment:
Could you add a reference to the relevant docs using `:ref:...` ?

Reply:
Please add new entries always on top of the list.

Reply:
Yes and of course.